### DISCUSSION TITLE
I have added slovenian translation to msg.py

### DISCUSSION THREAD

**Discussion by tomazfs** [israel-dryer/ttkbootstrap#503] [General]
I tried to open an issue, it didn't work. So I'm posting here.
Could you please integrate slovenian translation.

[msgs.zip](https://github.com/israel-dryer/ttkbootstrap/files/13067749/msgs.zip)

**Comment by TheDemonGuard** [comment]
@tomazfs 
Thank you for translating. I have created a PR #727 

> I tried to open an issue, it didn't work. So I'm posting here. Could you please integrate slovenian translation.
> 
> [msgs.zip](https://github.com/israel-dryer/ttkbootstrap/files/13067749/msgs.zip)
